src/api/messages.ts

```diff
@@ -21,3 +22,28 @@ export const postMessage = async (formData: {
     };
   }
 };
+
+export const getMessages = async (isApproved?: boolean | null) => {
+  const data =
+    (await apiGet<Message[]>(
+      `messages/email/received-owner-requests?approvedFilter=${isApproved}`,
+    )) || [];
+
+  return data;
```

Comment:
This is probably more readable: 

```suggestion
  const data =
    await apiGet<Message[]>(
      `messages/email/received-owner-requests?approvedFilter=${isApproved}`,
    )

  return data || []
```